scripts/clone_cloudevent_protos.dart

```diff
@@ -17,28 +17,32 @@ void setupLogging() {
   });
 }
 
-Future<void> cloneCloudEvents() async {
```

Comment:
nit: instead of a bool we can define a status construct for more descriptive errors.

Example 1:
```dart
enum MyStatus {
  ok,
  error,
}
```

Example 2:
```dart
sealed class Status {}

final class Ok<T> extends Status {
  Ok(this.data);

  final T data;
}

final class Failure extends Status {
  const Failure({this.message});

  final String message;
}
```